### PR TITLE
Feat/revise image component

### PR DESCRIPTION
**Pull request by pitb2022**

### CODE REVIEW

File: .env.dev [hunk 1]
```diff
@@ -7,7 +7,8 @@
 ###
 NEXT_PUBLIC_RUNTIME_ENV=development
 NEXT_PUBLIC_SITE_DOMAIN=web-develop.matters.news
-NEXT_PUBLIC_ASSET_DOMAIN=assets-develop.matters.news
+NEXT_PUBLIC_LEGACY_ASSET_DOMAIN=assets-develop.matters.news
```
Comment (robertu7) [inline .env.dev:10]: can also update to `.env.local.example`

File: src/common/utils/text/article.ts [hunk 2]
```diff
@@ -100,9 +100,8 @@ export const optimizeEmbed = (content: string) => {
         return /* html */ `
       <picture>
         <source
-          type="image/webp"
```
Comment (robertu7) [inline src/common/utils/text/article.ts:103]: L102-L106 are duplicated with L108-L112

File: src/components/ResponsiveImage/index.tsx [hunk 3]
```diff
@@ -35,12 +35,10 @@ const BaseResponsiveImage = ({
       {smUpSize && (
         <>
           <source
-            type="image/webp"
```
Comment (robertu7) [inline src/components/ResponsiveImage/index.tsx:38]: Same duplication issue as above.

**Review by robertu7 — CHANGES_REQUESTED** [review]

File: .env.dev (hunk 1 above)
Comment (pitb2022) [reply .env.dev:10]: done

File: src/common/utils/text/article.ts (hunk 2 above)
Comment (pitb2022) [reply src/common/utils/text/article.ts:103]: Removed

File: src/components/ResponsiveImage/index.tsx (hunk 3 above)
Comment (pitb2022) [reply src/components/ResponsiveImage/index.tsx:38]: Removed

**Pull request merged**